gaffer-as-a-service/README.md

```diff
@@ -24,7 +24,14 @@ To access the Eureka Dashboard when running locally visit `http://localhost:8761
   * Gaas-Rest
   * Eureka Server
   * Your chosen side car
-* In GaaS-UI set the ENV variables in the `.env` file. For example, when running everything using JWT authentication, the env variables are set to the image shown below:
-![image](https://user-images.githubusercontent.com/60354187/160592401-e0127719-56cb-4353-b3e9-f100f75a850c.png)
+* In GaaS-UI set the ENV variables in the `.env` file. For example, when running everything using JWT authentication, the env variables are set as below:
+* `REACT_APP_API_PLATFORM="OTHER"
```

Comment:
If you use triple backticks then it would look a lot nicer:
```
REACT_APP_API_PLATFORM="OTHER"
REACT_APP_KAI_REST_API_HOST="http://localhost:8081/gaas-rest-service/"
REACT_APP_COGNITO_USERPOOLID=""
REACT_APP_COGNITO_CLIENTID=""
REACT_APP_AUTH_ENDPOINT="http://localhost:8081/gaas-sidecar-service/"
REACT_APP_COGNITO_SCOPE = ""
REACT_APP_COGNITO_REDIRECT_URI = ""
SKIP_PREFLIGHT_CHECK="true"
```

Reply:
Done thanks